src/js/components/Menu/__tests__/__snapshots__/Menu-test.js.snap

```diff
@@ -2492,9 +2489,6 @@ exports[`Menu open and close on click 3`] = `
   border: none;
   padding: 0;
   text-align: inherit;
-  background-color: rgba(221,221,221, 0.4);
```

Comment:
Please, doubles check all these styles didn't exist prior to your previous changes as well. I want to make sure we are not introducing regression from the style aspect.

Reply:
I confirmed these styles didn't exist prior to my previous changes.